qds_sdk/cloud/oracle_bmc_cloud.py

```diff
@@ -0,0 +1,204 @@
+                              image_id=arguments.image_id,
+                              vcn_id=arguments.vcn_id,
+                              storage_tenant_id=arguments.storage_tenant_id,
+                              storage_user_id=arguments.storage_user_id,
+                              storage_key_finger_print=arguments.storage_key_finger_print,
+                              storage_api_private_rsa_key=arguments.storage_api_private_rsa_key)
+
+    def create_parser(self, argparser):
+
+        # compute settings parser
+        compute_config = argparser.add_argument_group("compute config settings")
+        compute_creds = compute_config.add_mutually_exclusive_group()
+        compute_creds.add_argument("--enable-account-compute-creds",
+                                   dest="use_account_compute_creds",
+                                   action="store_true",
+                                   default=None,
+                                   help="to use account compute credentials")
+        compute_creds.add_argument("--disable-account-compute-creds",
+                                   dest="use_account_compute_creds",
+                                   action="store_false",
+                                   default=None,
+                                   help="to disable account compute credentials")
+        compute_config.add_argument("--compute-tenant-id",
+                                    dest="compute_tenant_id",
+                                    default=None,
+                                    help="tenant id for oracle cluster")
+        compute_config.add_argument("--compute-user-id",
+                                    dest="compute_user_id",
+                                    default=None,
+                                    help="compute user id for oracle cluster")
+        compute_config.add_argument("--compute-key-finger-print",
+                                    dest="compute_key_finger_print",
+                                    default=None,
+                                    help="compute key fingerprint for oracle cluster")
+        compute_config.add_argument("--compute-api-private-rsa-key",
+                                    dest="compute_api_private_rsa_key",
+                                    default=None,
+                                    help="compute api private rsa key for oracle cluster")
+
+        # location settings parser
+        location_group = argparser.add_argument_group("location config settings")
+        location_group.add_argument("--oracle-region",
+                                    dest="region",
+                                    help="region to create the cluster in", )
+        location_group.add_argument("--oracle-availability-zone",
+                                    dest="availability_domain",
+                                    help="availability zone to" +
+                                         " create the cluster in", )
+
```

Comment:
This is done intentionally for separation of different parser for better visibility.